projects/components/src/form/form-select/form-select.component.ts

```diff
@@ -30,6 +30,9 @@ export class FormSelectComponent extends BaseFormControl {
         if (!this.options) {
             return undefined;
         }
-        return this.options.find(option => option.value === this.formControl.value);
+        // option.value is string | number. Also, formControl.value is string when the dropdown value is manually chosen
```

Comment:
I think the only comment you need is that the option may be of type number or string